pkg/fakerp/config.go

```diff
@@ -65,3 +77,76 @@ func NewConfig(log *logrus.Entry) (*Config, error) {
 	os.Setenv("AZURE_REGION", c.Region)
 	return &c, nil
 }
+
+func getPluginConfig(basePath string) (*api.PluginConfig, error) {
+	tc := api.TestConfig{
+		RunningUnderTest:      os.Getenv("RUNNING_UNDER_TEST") != "",
+		ImageResourceGroup:    os.Getenv("IMAGE_RESOURCEGROUP"),
+		ImageResourceName:     os.Getenv("IMAGE_RESOURCENAME"),
+		DeployOS:              os.Getenv("DEPLOY_OS"),
+		ImageOffer:            os.Getenv("IMAGE_OFFER"),
+		ImageVersion:          os.Getenv("IMAGE_VERSION"),
+		ORegURL:               os.Getenv("OREG_URL"),
+		EtcdBackupImage:       os.Getenv("ETCDBACKUP_IMAGE"),
+		AzureControllersImage: os.Getenv("AZURE_CONTROLLERS_IMAGE"),
+	}
+
+	// populate geneva artifacts
+	var cb, kb, db []byte
+	var crt *x509.Certificate
+	var key *rsa.PrivateKey
+	var err error
+	var syncImage string
+	artifactDir := fmt.Sprintf("%s/", basePath)
+	// if test files exist, load them
+	if fileExist(artifactDir+"logging-int.cert") &&
```

Comment:
Diff implementation in follow-up PR:
https://github.com/openshift/openshift-azure/pull/773/files#diff-cdf17c3f999df3ea27f3c7b5e0520ef5R94